### ISSUE TITLE
Animation bug when closing keyboard

### ISSUE THREAD

**Issue by psawa** [signalapp/Signal-iOS#5468]
- [ x ] I have searched open and closed issues for duplicates
- [  x ] I am submitting a bug report for existing functionality that does not work as intended
- [ x ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
When the keyboard is open in a conversation, the only way to close it is to swipe down. This induces a weird hiccup in the animation: An abrupt scroll (or jump) up, not smooth at all. 

### Steps to reproduce
- Open a conversation
- Click on the input field to open the keyboard
- Gently Swipe down to close the keyboard

**Actual result:** The conversation is abruptly scrolled up

**Expected result:** The keyboard should close, and the conversation should scroll smoothly until its takes over the keyboard space.

### Screenshots

Would need a video.

### Device info

**Device**: iPhone 13 Mini

**iOS version**: 16.0.3

**Signal version:** 5.58.0.12

**Comment by mcsitter** [comment]
I could not reproduce the hiccup you were describing. Maybe you could share a screen recording?

**Comment by EvanHahn-Signal** [comment]
I couldn't reproduce either. Could you share a screencast?

**Comment by Doge7734** [comment]
I’m able to reproduce this on the beta.

iOS version: 16.1.2
Signal version: 6.4.0.8
Device: iPhone 12 Mini

Debug log: https://debuglogs.org/ios/6.4.0/2bb2daa791eb10ae4b86c10eec74f66fa931e1df4f5560d15213e4bb17f72e78.zip

**Comment by EvanHahn-Signal** [comment]
@Doge7734 Could you take a screencast? I'd love to see what the bug looks like for you.

**Comment by Doge7734** [comment]
Yep, I can do that. I recorded a  video before but wasn't able to upload it via the web on my phone so I have just downloaded the app.

Recording: https://user-images.githubusercontent.com/101482874/204950977-1fa077dd-2672-4698-8e41-cdac0872b6db.MOV

**Comment by EvanHahn-Signal** [comment]
@Doge7734 Thanks! I've reproduced this and filed an issue in our internal tracker.
